<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generalContent</class>
 <widget class="GeneralContent" name="generalContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelNumberType">
           <property name="text">
            <string>TBD Nummer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNr">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPos"/>
         </item>
         <item>
          <widget class="QLabel" name="labelPosError">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Artikel-Nummer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editArtNr"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Haupt-Text:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editMainText"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Extra-Text:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="editText">
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Funktionen</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Material (F1)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Leistung (F2)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Neuaufnahme Material (F5)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Neuaufnahme Leistung (F6)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Aktuelle Position:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Allgemeiner Stundensatz</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="labelGenRate">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>EUR</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>EUR</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="labelProfitMatEur">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="labelProfitMatPerc">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Gewinn aus Material</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Arbeitszeit ges.</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="labelWorkingHours">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Gewinn aus Material.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelTotalType">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>TBD gesamt:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Gewinn aus Material</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="labelProfitTotalPerc">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Gewinn aus Material</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>EUR</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="labelProfitTotalEur">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="16" column="1">
      <widget class="QLabel" name="label_55">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="label_58">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="editMaterialEKP"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Aufschlag</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_54">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>Hilfsmaterial</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_59">
       <property name="text">
        <string>Einzelpreis</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_42">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Leistung</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Sonstiges</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="labelPriceTotal">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLineEdit" name="editServiceTime"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>Material</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="editMaterialPrice"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="editMaterialSurchage"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>Rabatt</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>Zeit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_30">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Material</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="editMaterialDiscount">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLineEdit" name="editServicePrice"/>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>Stundensatz</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>EUR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="editUnitType"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="labelMaterialQuant">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="editUnitSize"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Menge:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLineEdit" name="editHelpMat"/>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_62">
       <property name="text">
        <string>Gesamtpreis</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLineEdit" name="editServiceRate"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>EKP</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>Material nach Rabatt:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="labelEP">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_27">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Kalkulation: Listenpreis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Mengeneinheit:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Zusammenfassung</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralContent</class>
   <extends>QWidget</extends>
   <header>pages/general_page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editPos</tabstop>
  <tabstop>editArtNr</tabstop>
  <tabstop>editMainText</tabstop>
  <tabstop>editText</tabstop>
  <tabstop>editUnitType</tabstop>
  <tabstop>editUnitSize</tabstop>
  <tabstop>editMaterialEKP</tabstop>
  <tabstop>editMaterialSurchage</tabstop>
  <tabstop>editMaterialPrice</tabstop>
  <tabstop>editMaterialDiscount</tabstop>
  <tabstop>editServiceRate</tabstop>
  <tabstop>editServiceTime</tabstop>
  <tabstop>editServicePrice</tabstop>
  <tabstop>editHelpMat</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>CopyData(QString)</slot>
  <slot>TakeFromMaterial()</slot>
  <slot>TakeFromService()</slot>
  <slot>MakeNewEntry()</slot>
 </slots>
</ui>
